src/engine.rs

```diff
@@ -241,3 +246,8 @@ fn deserialize_message(data: Vec<u8>, topic: Topic) -> Result<Message, std::io::
         DroppedTxns => DroppedTxnMessage::deserialize(&mut data).map(Message::DroppedTxns),
     }
 }
+
+/// Convert [std::net::Ipv4Addr] and port [u16] into MultiAddr[libp2p::Multiaddr] type
```

Comment:
/// Convert **ip address**[std::net::Ipv4Addr] and port [u16] into MultiAddr[libp2p::Multiaddr] type